modeling-cmds/src/def_enum.rs

```diff
@@ -951,6 +958,14 @@ define_modeling_cmd_enum! {
             pub padding: f32,
         }
 
+        /// Fit the view to the scene with an isometric view.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct ViewIsometric {
```

Comment:
So this zooms out to fill all objects within the scene?

Reply:
this was Serena's PR that already has merged into main.  again, read my top comment about the diff.